data/cha_130540-773958.json

```diff
@@ -14,6 +14,25 @@
     "Names": [
         {
             "other_name": "Cha 130540-773958"
+        },
+        {
+            "other_name": "Cha J1305-7739"
+        }
+    ],
+    "Spectra": [
+        {
+            "spectrum": "https://bdnyc.s3.amazonaws.com/ISAAC/ChaJ1305_ISAACspec_JHK_nocosmics_2_dered.fits",
+            "local_spectrum": null,
+            "regime": "nir",
+            "telescope": "ESO VLT U3",
+            "instrument": "ISAAC",
+            "mode": "SW LRes",
+            "observation_date": "2010-04-21T00:00:00",
+            "wavelength_units": null,
+            "flux_units": null,
+            "wavelength_order": null,
+            "comments": "April 6, 2010",
```

Comment:
The comment with an additional observation date should have more words. For example, "spectrum also includes data observed on April 6, 2010."

Reply:
I made the comment more comprehensive in the new csv.